_data/icons/doken.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafkreifms4eio6v56oyeemnnu5luq3sc44hptan225lr45itgzu3u372iu",
+    "width": 1024,
+    "height": 1024,
+    "format": "png"
```

Comment:
identify bafkreifms4eio6v56oyeemnnu5luq3sc44hptan225lr45itgzu3u372iu
bafkreifms4eio6v56oyeemnnu5luq3sc44hptan225lr45itgzu3u372iu PNG 200x200 200x200+0+0 8-bit sRGB 28797B 0.000u 0:00.000

Reply:
Should i resize the image or just change the width and height in json as the corresponding image size?

Reply:
I just updated the json to 200 x 200